docs/source/tutorials/time.qmd

```diff
@@ -10,9 +10,9 @@ The fundamental time unit should represent a period of fixed (or approximately f
 
 For many infectious disease renewal models of interest, the fundamental time unit will be days, and we will proceed with this tutorial treating days as our fundamental unit.
 
- `pyrenew` deals with time having `RandomVariable`s carry information about (i) their own time unit expressed relative to the fundamental unit (`t_unit`) and (ii) the starting time, `t_start`, measured relative to `t = 0` in model time in fundamental time units.
+`pyrenew` deals with time having `RandomVariable`s carry information about (i) their own time unit expressed relative to the fundamental unit (`t_unit`) and (ii) the starting time, `t_start`, measured relative to `t = 0` in model time in fundamental time units. Return values from `RandomVariable.sample()` are `tuples` or `namedtuple`s of `SampledValue` objects. Each such `SampledValue` is optionally time-aware with specifiable `t_start` and `t_unit` attributes.
```

Comment:
`RandomVariable`s having their own `t_start` and `t_unit` seems to contrast with the idea the `RandomVariable.sample()` can return multiple `SampledValue`s with different `t_start` and `t_unit`.

Reply:
Attempted clarification in 859dbc9: 

https://github.com/CDCgov/multisignal-epi-inference/blob/859dbc96712a887bf2a5daa98156bebbedbc8dab/docs/source/tutorials/time.qmd#L13-L15

Reply:
In the longer run, we should add an example of the second thing to the tutorial, but I think that can wait for the next time-handling PR.